lib/sidekiq/cron/job.rb

```diff
@@ -432,6 +432,12 @@ def to_hash
 
         if date_as_argument?
           hash.merge!(date_as_argument: "1")
+        else
```

Comment:
Instead of this, we could just assign a value similar to others:

E.g.
```ruby
hash = {
  ...
  date_as_argument: (date_as_argument? ? "1" : "0")
}
```

So we don't need `if date_as_argument?` condition at all.

Reply:
@freemanoid Please have a look at this comment https://github.com/sidekiq-cron/sidekiq-cron/pull/485#issuecomment-2351677551